### ISSUE TITLE
aria-label for DropdownButton

### ISSUE THREAD

**Issue by sernaferna** [react-bootstrap/react-bootstrap#6768]
### Is your feature request related to a problem? Please describe

Lighthouse is raising an accessibility issue for a `DropdownButton` that doesn't have an accessible name, but giving it an `aria-label` isn't having the desired effect. e.g.

```lang-tsx
<DropdownButton
  title={<FolderSymlink />}
  variant="outline-secondary"
  aria-label="Move Item"
>
  {listIds.map((i) => (
    <Dropdown.Item key={i.id}>{i.label}</Dropdown.Item>
  ))}
</DropdownButton>
```

Tthe button doesn't have any text, just an SVG icon. With a regular button the `aria-label` attribute would take care of that warning.

### Describe the solution you'd like

If `aria-label` is set for the `DropdownButton` it should get set on the underlying `button` -- in addition to or instead of the surrounding `div`, whatever makes the most sense.

### Describe alternatives you've considered

I did also try setting `menuRole` to `menu` but the Lighthouse error doesn't go away.

### Additional context

The `aria-label` *is* getting set on the surrounding `div`, but not on the button itself. So it's generating something like:

```lang-html
<div aria-label="Move Item" class="dropdown">
  <button type="button" aria-expanded="false" class="dropdown-toggle btn btn-outline-secondary">
    <svg/>
  </button>
</div>
```

**Comment by kyletsang** [comment]
For your use case, it's better to follow [this example](https://react-bootstrap.github.io/docs/components/dropdowns#single-button-dropdowns), since it will give you more control of what attributes go where.  DropdownButton is something that handles most cases that doesn't need too much customization

**Comment by sernaferna** [comment]
Can confirm this works, so I'll close the issue.

I have to ask, though: is accessibility really something that's considered "too much customisation" for DropdownButton? This is something that only causes accessibility issues when you're using an icon instead of text for the button, but that feels like it might be a common enough case that DropdownButton should be able to handle it...
